### ISSUE TITLE
Command PhaseScriptExecution failed with a nonzero exit code

### ISSUE THREAD

**Issue by rlee1990** [flutter/flutter#72726]
I am receiving the below error when running a build for a flutter app on IOS with a physical device on a mac M1 chip set. This was working fine on Saturday 12/19/20.

/Users/rickeylee/Desktop/flutter/packages/flutter/lib/material.dart:57:1: Error: Error when reading '/Users/rickeylee/Desktop/flutter/packages/flutter/lib/src/material/date_picker.dart': No such file or directory
export 'src/material/date_picker.dart';
^

Command PhaseScriptExecution failed with a nonzero exit code

[✓] Flutter (Channel dev, 1.26.0-1.0.pre, on macOS 11.0 20A2411 darwin-arm,
    locale en-US)
    • Flutter version 1.26.0-1.0.pre at /Users/rickeylee/Desktop/flutter
    • Framework revision 63062a6443 (8 days ago), 2020-12-13 23:19:13 +0800
    • Engine revision 4797b06652
    • Dart version 2.12.0 (build 2.12.0-141.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/rickeylee/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.17.0

[✓] Connected device (2 available)
    • Rick (mobile) • 00008101-0011542C3A00001E • ios            • iOS 14.3
    • Chrome (web)  • chrome                    • web-javascript • Google Chrome
      87.0.4280.88

• No issues found!

**Comment by rlee1990** [comment]
I can confirm that just creating a new flutter application generates the same error.

**Comment by rlee1990** [comment]
@ianloic

**Comment by rlee1990** [comment]
@TahaTesser @fluttergithubbot

**Comment by chunhtai** [comment]
cc @christopherfujino

**Comment by rlee1990** [comment]
@chunhtai is there anyone that can look into this. I can't release an update to an app due to this.

**Comment by chunhtai** [comment]
can you double check if this file exist at all?
`/Users/rickeylee/Desktop/flutter/packages/flutter/lib/src/material/date_picker.dart`

**Comment by rlee1990** [comment]
@chunhtai  after checking it only goes to /Users/rickeylee/Desktop/flutter/packages/flutter/lib/src/material.dart

**Comment by rlee1990** [comment]
Could it be the Flutter package was messed up when upgrading?

**Comment by chunhtai** [comment]
@rlee1990 it maybe did, can you try to reinstall flutter?

**Comment by rlee1990** [comment]
@chunhtai will try that again now and see what happens

**Comment by rlee1990** [comment]
So I redownloaded Flutter from the site and checked it does not have date_picker.dart at /Users/rickeylee/Desktop/flutter/packages/flutter/lib/src/material/date_picker.dart @chunhtai

**Comment by rlee1990** [comment]
<img width="1080" alt="Screen Shot 2020-12-21 at 2 47 47 PM" src="https://user-images.githubusercontent.com/21372502/102816185-998bcd00-439b-11eb-8e4b-17d41d5ddd6f.png">

**Comment by chunhtai** [comment]
That is quite weird, cc @darrenaustin who has been working on the date_picker

**Comment by rlee1990** [comment]
@chunhtai thanks. I even checked on my Windows device and it's the same thing I am going to try to see if I pull an older version and update it if this still happens.

**Comment by rlee1990** [comment]
So I checked it is showing on the master branch when looking on Github trying to install it from there

**Comment by rlee1990** [comment]
@chunhtai I can confirm cloning the repo from Github for the master channel works. So the issue is with the packages on the Flutter.dev site

**Comment by chunhtai** [comment]
The datepicker is not yet in stable. If you download the zip from the dev channel directly, you can see the file is there https://storage.googleapis.com/flutter_infra/releases/dev/windows/flutter_windows_1.26.0-1.0.pre-dev.zip

I think there maybe something wrong when you switch the channel from an existing sdk

**Comment by rlee1990** [comment]
Thanks, I will close this as it is now working when I cloned flutter from the master branch @chunhtai

**Comment by darrenaustin** [comment]
Thanks @chunhtai for helping @rlee1990 resolve this issue. The date picker internal code was restructured a while back which moved some files around. This change may not have migrated to stable yet. Not sure what got messed up during your upgrade though. Glad you are up and working again.

**Comment by greg-fischer** [comment]
I just want to leave notes here even though this is solved, but I ran into this same issue after upgrading to 1.26.x.  For me it was one of several issues that didn't come up until I ran a build for chrome, which showed several new build errors.  I don't use DatePicker.
- something in analysis_options.yaml was invalid - deleted contents.
- a new folder for integration_tests showed up with errors.  deleted.
- a file in test/ showed up with errors about "MyApp()", deleted.
- some of my libraries had an error about "path", and I needed publish_to: 'none' added to pubspec.
- I updated all the packages in pubspec.yaml

Both macOS and iOS were failing to build and I never saw a "show errors" popup in VSCode, even with new project platform folders created. iOS gave the error posted in terminal.  Chrome did allow me to show the errors and would build once these new issues were fixed, and then iOS and macOS built fine.
